diff --git a/Sources/AppBundle/model/Rect.swift b/Sources/AppBundle/model/Rect.swift
--- a/Sources/AppBundle/model/Rect.swift
+++ b/Sources/AppBundle/model/Rect.swift
@@ -66,7 +66,7 @@ extension Rect {
     var nsRect: NSRect {
         NSRect(
             x: topLeftX,
-            y: mainMonitor.height - topLeftY - height,
+            y: mainMonitorInfo.height - topLeftY - height,
             width: width,
             height: height,
         )
